File: python/iniciante/problem1013.py
# Inputs

a, b, c = input('Enter three numbers: ').split()
a = int(a)
b = int(b)
c = int(c)

# Calculations

maiorab = (a + b + abs(a - b))/2
output = (maiorab + c + abs(maiorab - c))/2

# Prints

print('%d eh o maior' % output)

File: go/01-iniciante/problem1013.go
package main

import (
    "fmt"
    "math"
)

func main() {
    var a, b, c int

    fmt.Scanf("%d %d %d", &a, &b, &c)

    maiorab := (float64(a)+float64(b) + math.Abs(float64(a)-float64(b)))/2
    maiorabc := (float64(maiorab)+float64(c) + math.Abs(float64(maiorab) - float64(c)))/2
    output := fmt.Sprintf("%.f", maiorabc)

    fmt.Println(output, "eh o maior")

}
